build.cake

```diff
@@ -796,72 +888,91 @@ string GetNuGetPackage(string packageId, string packageVersion)
     return filename;
 }
 
-void ExtractNuGetPackage (IPackage package, FrameworkName frameworkName, string tempContentPath, string destination) {
-    if (package != null) {
-        IEnumerable<IPackageAssemblyReference> assemblies;
-        VersionUtility.TryGetCompatibleItems (frameworkName, package.AssemblyReferences, out assemblies);
-
-        // Move assemblies.
-        foreach (var i in assemblies) {
-            if (!FileExists (destination + "/" + i.Name)) {
-                MoveFile (tempContentPath + "/" + i.Path, destination + "/" + i.Name);
-            }
-        }
-
+FilePathCollection ResolveDllFiles(string tempContentPath) 
+{
+    var uwpPath = $"{tempContentPath}/lib/{UwpFrameworkName.GetShortFolderName()}/*.dll";
+    var files = GetFiles(uwpPath);
+    if (files.Any()) 
+    {
+        return files;
+    }
+    var netStandard2Path = $"{tempContentPath}/lib/{Netstandard2FrameworkName.GetShortFolderName()}/*.dll";
+    files = GetFiles(netStandard2Path);
+    if (files.Any()) 
+    {
+        return files;
+    }
+    if (files.Any()) 
+    {
+        return files;
     }
-    // Move native binaries.
-    var contentPathSuffix = "lib/uap10.0/";
+    Warning($"Haven't found anything under {tempContentPath} - make sure it's expected.");
+    return null;
+}
+
+string MoveNativeBinaries(string tempContentPath, string destination)
+{
     var runtimesPath = tempContentPath + "/runtimes";
     if (DirectoryExists (runtimesPath)) {
+        Console.WriteLine("MoveNativeBinaries");
         var oneArch = "x86";
         foreach (var runtime in GetDirectories (runtimesPath + "/win10-*")) {
             var arch = runtime.GetDirectoryName ().ToString ().Replace ("win10-", "").ToUpper ();
             oneArch = arch;
-            var nativeFiles = GetFiles (runtime + "/native/*");
+            var nativeFiles = GetFiles (runtime + "/**/*.dll");
```

Comment:
Why did you change it?

Reply:
@russelarms

Reply:
The script looked for files in locations like `externals/uwp/SQLitePCLRaw.lib.e_sqlite3/runtimes/win10-arm64/native/*`
But now files for win10 are located in
`externals/uwp/SQLitePCLRaw.lib.e_sqlite3/runtimes/win10-arm64/nativeassets/uap10.0/e_sqlite3.dll`
So it doesn't work without that change.